### PR TITLE
DLP: Added sample for redacting image with color coded infotypes

### PR DESCRIPTION
**Pull request by jainamtrivedi-crest**
- Added sample for [Redact Image with Color Coded Infotypes](https://cloud.google.com/dlp/docs/samples/dlp-redact-image-colored-infotypes) and test cases

### CODE REVIEW

File: dlp/api/Snippets.Tests/resources/test_redact_image_colored.png
Comment (annadata) [inline dlp/api/Snippets.Tests/resources/test_redact_image_colored.png:1]: There was a comment earlier to not use trade marks in samples.
Please use the image from https://github.com/GoogleCloudPlatform/golang-samples/pull/3149/files#diff-3647b36a39da2823adf7be804c9a76fc2bb417b5a0a2fe988bb75d53920ad6ea

File: dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+        // Call the API.
+        RedactImageResponse response = dlp.RedactImage(request);
+
+        // Inspect the response.
+        Console.WriteLine($"Redacted image written to: {redactedImagePath}");
+
+        // Writes redacted image into file
+        response.RedactedImage.WriteTo(new FileStream(redactedImagePath, FileMode.Create, FileAccess.Write));
+
```
Comment (annadata) [inline dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs:90]: Please write file first then the statement.

**Review by annadata — APPROVED** [review]

File: dlp/api/Snippets.Tests/resources/test_redact_image_colored.png
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets.Tests/resources/test_redact_image_colored.png:1]: Done. Thanks

File: dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs (hunk 1 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs]: Done. Thanks

File: dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+            ByteItem = byteContentItem
+        };
+
+        // Call the API.
+        RedactImageResponse response = dlp.RedactImage(request);
+
+        // Writes redacted image into file
+        response.RedactedImage.WriteTo(new FileStream(redactedImagePath, FileMode.Create, FileAccess.Write));
+
+        // Inspect the response.
```
Comment (amanda-tarafa) [inline dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs:88]: nit: This is really not inspecting the response
```suggestion
```

File: dlp/api/Snippets.Tests/RedactImageWithColorCodedInfoTypesTests.cs [hunk 3]
```diff
@@ -0,0 +1,37 @@
+﻿
+
+
```
Comment (amanda-tarafa) [inline dlp/api/Snippets.Tests/RedactImageWithColorCodedInfoTypesTests.cs:3]: nit: remove blank lines

**Review by amanda-tarafa — COMMENTED** [review]
Two tiny nits, otherwise looks good.

File: dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs (hunk 2 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets/RedactImageWithColorCodedInfoTypes.cs]: Done

File: dlp/api/Snippets.Tests/RedactImageWithColorCodedInfoTypesTests.cs (hunk 3 above)
Comment (jainamtrivedi-crest) [reply dlp/api/Snippets.Tests/RedactImageWithColorCodedInfoTypesTests.cs]: Done

**Pull request merged**